### DISCUSSION TITLE
What icons are available, or are shipped with beercss?

### DISCUSSION THREAD

**Discussion by janat08** [beercss/beercss#304] [Q&A] [answered]
There seem to be some icons that get shipped with beercss, whats the list of them? I'm struggling to download material icons one by one, it just downloads almost MB of icons.

**Comment by leonardorafael** [comment, marked as answer]
Hi,

The local icon fonts (.woff2) are from the urls:

https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:FILL@0..1
https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:FILL@0..1
https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:FILL@0..1
